examples/app-pytorch/client_new.py

```diff
@@ -0,0 +1,24 @@
+import flwr
+from flwr.common import Message, Context
+
+
+# Run via `flower-client-app client:app`
+app = flwr.client.ClientApp()
+
+
+@app.train()
+def train(msg: Message, ctx: Context):
+    print("`train` is not implemented, echoing original message")
+    return msg.create_reply(msg.content, ttl=msg.metadata.ttl)
+
+
+@app.evaluate()
+def eval(msg: Message, ctx: Context):
+    print("`evaluate` is not implemented, echoing original message")
+    return msg.create_reply(msg.content, ttl=msg.metadata.ttl)
+
+
+@app.query()
+def q(msg: Message, ctx: Context):
```

Comment:
```suggestion
def query(msg: Message, ctx: Context):
```
? although it could be left as `q`.

Reply:
i named it `q` to show that the function can have a different name than the decorator, but we can rename it to `query` for consistency. we should probably give it longer name anyways (sth that has more than one character)